present.md

```diff
@@ -1,11 +1,9 @@
-This section focuses on more current developments in Faker JS (hereafter referred to as simply Faker).
+The present version of Faker differs from the past in several key areas. First, the folder structure has changed and now includes a source and documentation folder, in addition to several integrations that allow Faker to be automated.
 
-Let us start by looking at version 6 of Faker, which is demarked by the project being rewritten in typescript, adding documentation, setting up a CI/CD pipeline, and beginning to make major bug fixes regarding the implementation.
-Towards the end of version 6, new features such as random IMEI numbers and MongoDB ObjectId generation were added. 
+More recent versions of Faker, starting at version 6, demarked by being rewritten in typescript, have new features such as better documentation, setting up a CI/CD pipeline, and an increased effort to fix major bugs within the project. Towards the end of version 6, new features such as random IMEI numbers and MongoDB ObjectId generation were added.
 
-Version 7 saw the Faker team overhauling the project again with continual code cleanup and improvements. 
-The ground was broken by removing code that had become deprecated, targeting es2020, and dropping support for Node v12, amongst other things. 
+Version 7 saw the Faker team overhauling the project again with continual code cleanup and improvements. The ground was broken by removing code that had become deprecated, targeting es2020, and dropping support for Node v12, amongst other things.
 
-As of version 7.5, most changes have centered around improving documentation, removing unused code, reducing code duplication, updating dependencies, and adding highly requested new features. 
+As of version 7.5, most changes have centered around improving documentation, removing unused code, reducing code duplication, updating dependencies, and adding highly requested new features.
 
-Currently, Faker sits at version 7.6 and has begun making major preparations for its version 8 release, in which its modules will be refactored to better support tree shaking. 
+Currently, Faker sits at version 7.6 and has begun making major preparations for its version 8 release, in which its modules will be refactored to better support tree shaking.
```

Comment:
```suggestion
Currently, Faker sits at version 7.6 and has begun making major preparations for its version 8 release, in which its modules will be refactored and reorganized.
```

We talk plenty about tree shaking in the next section. I don't think it needs a special mention here.